polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -1496,6 +1501,34 @@ struct GPUReturnOpLowering : public ConvertOpToLLVMPattern<gpu::ReturnOp> {
   }
 };
 
+struct EraseSPIRVBuiltinPattern
+    : public OpRewritePattern<spirv::GlobalVariableOp> {
+  using OpRewritePattern<spirv::GlobalVariableOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(spirv::GlobalVariableOp global,
+                                PatternRewriter &rewriter) const final {
+    auto *parent = global->getParentWithTrait<OpTrait::SymbolTable>();
+    if (!parent)
+      return failure();
+    const auto name = global.getName();
+    auto &region = parent->getRegion(0);
+    bool alreadyDefined =
+        std::any_of(region.begin(), region.end(), [&](auto &block) {
```

Comment:
Can you use llvm::any_of here and in similar places.

Reply:
Done!